cmd/ipfs/init.go

```diff
@@ -238,12 +238,12 @@ func identityConfig(nbits int) (config.Identity, error) {
 		return ident, debugerror.New("Bitsize less than 1024 is considered unsafe.")
 	}
 
-	fmt.Printf("generating key pair...")
+	fmt.Printf("Generating %v-bit RSA keypair...", nbits)
 	sk, pk, err := ci.GenerateKeyPair(ci.RSA, nbits)
 	if err != nil {
 		return ident, err
 	}
-	fmt.Printf("done\n")
+	fmt.Printf(" Done\n")
```

Comment:
sorry to nitpick, i do want these to be lowercase. most unix commands are all strictly lowercase.